### DISCUSSION TITLE
求加一个一键导出b站观看历史记录的功能

### DISCUSSION THREAD

**Discussion by Rcrusade** [the1812/Bilibili-Evolved#2582] [功能建议 / Ideas]
我找到了类似方法 https://zhuanlan.zhihu.com/p/55484060, 奈何技术不达标不能实现，特来求大佬实现，b站历史记录只能保存大概2个月，对于如今信息流爆炸无法追溯有价值信息的同学太不友好，如果能够线下保存数据，通过BV号或视频标题或视频缩略图反查，就能带来极大便利。

**Comment by the1812** [comment]
你要啥格式, json?

**Reply by Rcrusade** [reply]
不敢奢求太多，能把bv号和与之对应的视频标题导出为txt都行，毕竟小弟代码基础为0

**Comment by freemedom** [comment]
大概保存两个多月3个月

**Comment by freemedom** [comment]
可以参考这个方法

打开个人历史记录
下滑页面到底（点击鼠标中键自动下滑）
右键->另存为 或按Ctrl+S 保存到本地计算机
用浏览器打开保存的html文件
没了 作者：xnnswjndzz https://www.bilibili.com/read/cv21104305/ 出处：bilibili

**Comment by freemedom** [comment]
如果有人有导成json csv的格式需求的话我再写写代码
